### PR TITLE
docs: Small ADR-003 changes

### PR DESCRIPTION
**Pull request by toteki**
## Description

Small changes to ADR-003's message types and keeper keys as discussed during initial implementation.

----

### Author Checklist

*All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues.*

I have...

- [x] included the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] added `!` to the type prefix if API or client breaking change
- [ ] targeted the correct branch (see [PR Targeting](https://github.com/umee-network/umee/blob/main/CONTRIBUTING.md#pr-targeting))
- [ ] provided a link to the relevant issue or specification
- [ ] added a changelog entry to `CHANGELOG.md`
- [ ] included comments for [documenting Go code](https://blog.golang.org/godoc)
- [ ] updated the relevant documentation or specification
- [ ] reviewed "Files changed" and left comments if necessary
- [ ] confirmed all CI checks have passed

### Reviewers Checklist

*All items are required. Please add a note if the item is not applicable and please add
your handle next to the items reviewed if you only reviewed selected items.*

I have...

- [ ] confirmed the correct [type prefix](https://github.com/commitizen/conventional-commit-types/blob/v3.0.0/index.json) in the PR title
- [ ] confirmed `!` in the type prefix if API or client breaking change
- [ ] confirmed all author checklist items have been addressed
- [ ] reviewed state machine logic
- [ ] reviewed API design and naming
- [ ] reviewed documentation is accurate
- [ ] reviewed tests and test coverage
- [ ] manually tested (if applicable)

### CODE REVIEW

**Comment by toteki** [comment]
Putting the ADR change as a separate PR, so it can hit main first and the feature PR stays clean.

File: docs/architecture/ADR-003-borrow-assets.md [hunk 1]
```diff
@@ -105,9 +116,11 @@ Assuming a placeholder token allow-list of at least two elements (e.g. `uumee`,`
 ### Positive
 - uTokens used as collateral increase in base asset value in the same way that lend positions do. This counteracts borrow position interest.
 - UX of enabling/disabling token types as collateral is simpler than depositing specific amounts
+- lengthPrefixed(borrowerAddress) | tokenDenom key pattern facilitates sorting open positions by user
 
 ### Negative
 - `x/bank` module must be extended to prohibit peer-to-peer transfers of uTokens when they would violate `x/leverage` borrowing limit.
+- lengthPrefixed(borrowerAddress) | tokenDenom key pattern makes it more difficult to sort all open positions by token type.
```
Comment (alexanderbez) [inline docs/architecture/ADR-003-borrow-assets.md:123]:
```suggestion
- `lengthPrefixed(borrowerAddress) | tokenDenom` key pattern makes it more difficult to get all open borrow positions by token denomination.
```

File: docs/architecture/ADR-003-borrow-assets.md [hunk 2]
```diff
@@ -105,9 +116,11 @@ Assuming a placeholder token allow-list of at least two elements (e.g. `uumee`,`
 ### Positive
 - uTokens used as collateral increase in base asset value in the same way that lend positions do. This counteracts borrow position interest.
 - UX of enabling/disabling token types as collateral is simpler than depositing specific amounts
+- lengthPrefixed(borrowerAddress) | tokenDenom key pattern facilitates sorting open positions by user
 
 ### Negative
```
Comment (alexanderbez) [inline docs/architecture/ADR-003-borrow-assets.md:121]:
```suggestion
### Negative

```

File: docs/architecture/ADR-003-borrow-assets.md [hunk 3]
```diff
@@ -105,9 +116,11 @@ Assuming a placeholder token allow-list of at least two elements (e.g. `uumee`,`
 ### Positive
```
Comment (alexanderbez) [inline docs/architecture/ADR-003-borrow-assets.md:116]:
```suggestion
### Positive

```

File: docs/architecture/ADR-003-borrow-assets.md [hunk 4]
```diff
@@ -105,9 +116,11 @@ Assuming a placeholder token allow-list of at least two elements (e.g. `uumee`,`
...
 - uTokens used as collateral increase in base asset value in the same way that lend positions do. This counteracts borrow position interest.
 - UX of enabling/disabling token types as collateral is simpler than depositing specific amounts
+- lengthPrefixed(borrowerAddress) | tokenDenom key pattern facilitates sorting open positions by user
```
Comment (alexanderbez) [inline docs/architecture/ADR-003-borrow-assets.md:119]:
```suggestion
- `lengthPrefixed(borrowerAddress) | tokenDenom` key pattern facilitates getting open borrow positions by account address.
```

**Review by alexanderbez — APPROVED** [review]

**Pull request merged**